### ISSUE TITLE
btcd mishandles witness size checking #1

### ISSUE THREAD

**Issue by danbryan** [DefiantLabs/cosmos-tax-cli#295]
https://github.com/DefiantLabs/cosmos-tax-cli/security/dependabot/1

![image](https://user-images.githubusercontent.com/807940/215622418-13878c4a-d455-47ee-88de-362410993a29.png)

@esaari1 or @KyleMoser can we update https://github.com/DefiantLabs/cosmos-tax-cli/blob/main/go.mod#L43 to this new version?

**Comment by esaari1** [comment]
@KyleMoser can you try this? I updated version to 0.23.2. Get an error cause the subpackage btcec is not being downloaded. Get the same problem when I tried 0.23.4, which is the latest version. 

Looking on github.com, I do not even see anything newer that v0.22.1 for btcd. But there is a 0.23.4 if you look at
https://pkg.go.dev/github.com/btcsuite/btcd

and there you can see btcec. So I am not sure what is going on here with go mod.

**Comment by danbryan** [comment]
yeah, I would say try to get it to build with the patched version of btcd.

**Comment by esaari1** [comment]
So we cannot do this just yet. cosmos-sdk depends on btcd and does not have an update yet that is compatible with btcd version 2.32.x.

**Comment by danbryan** [comment]
@esaari1 thanks for looking into this.  We can move this back to todo, and try again later once the update is available.
